corehq/apps/data_cleaning/tasks.py

```diff
@@ -38,7 +38,7 @@ def commit_data_cleaning(self, bulk_edit_session_id):
     session.update_result(0)
     count_cases = case_load_counter("bulk_case_cleaning", session.domain)
 
-    record_iter = session.records.order_by('pk').iterator()
```

Comment:
Are we sure we don't need ordering here? Is it possible for the user to add or delete records for this session while this task is running?

Reply:
I'm pretty sure it isn't necessary

> Is it possible for the user to add or delete records for this session while this task is running?
It isn't possible, the session will be locked down after it is committed